app/scheduleSummaryReport/services/scheduleSummaryReportStateService.js

```diff
@@ -104,6 +112,32 @@ class ScheduleSummaryReportStateService {
 							sections.list[slotSection.id] = slotSection;
 						});
 
+
+						action.payload.supportAssignments.forEach( function(supportAssignment) {
+							if (supportAssignment.appointmentType != "teachingAssistant" || !supportAssignment.supportStaffId) { return; }
```

Comment:
Please add some comments to this section. It's not immediately clear to the reader why the data is organized this way.